service/settings.py

```diff
@@ -264,7 +264,7 @@ def getInstance(cls):
     def __init__(self):
         serviceHTMLExportDefaultSettings = {
             "enabled": False,
-            "path"   : config.pyfaPath + os.sep + 'pyfaFits.html',
+            "path"   : os.path.join(config.savePath, 'pyfaFits.html'),
```

Comment:
Could you elaborate on this please. It seems to be a Windows related issue and I can't reproduce.